calibration/step-2-measure-scales.py

```diff
@@ -45,15 +126,18 @@ def generate_responses(llm, input_batch, args):
     parser.add_argument("--max-num-prefill-seqs", type=int, default=1)
     parser.add_argument("--block-quant", action="store_true", default=False)
     parser.add_argument("--expert-parallel", action="store_true", default=False)
+    parser.add_argument("--auto-process-dataset", action="store_true", default=False)
```

Comment:
Suggest to add description on the purpose of these 3 new argument, especially auto-process-dataset.